### DISCUSSION TITLE
Define Marginal Carbon Intensity

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#178] [SCI feedback]
"This is the emissions intensity of the marginal power plant which will be turned up if you schedule some compute (e.g. increase electricity demand from the grid) at that moment." - this is vague and needs proper definition. What is a "marginal power plant"?

**Comment by atg-abhishek** [comment]
@Henry-WattTime would you like to add a clarification for this? Thanks!
